### ISSUE TITLE
Documenting our preview generation process

### ISSUE THREAD

**Issue by nullishamy** [catppuccin/toolbox#148]
If not done already, it would be nice to document how exactly to generate previews. 

This could provide links to docs on using an editor like GIMP to create each flavor, show how to use imagemagick (or similar) to convert those to webp (if needed), along with describing a simple catwalk command.

If this does exist, it would be good to put a link to it in the catwalk readme.

**Comment by uncenter** [comment]
We've been recommending the usage of `cwebp` for PNG -> WebP conversion over at the userstyles repository FYI: https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-convert-preview-images-to-webp.

**Comment by nullishamy** [comment]
Any particular reason?

**Comment by uncenter** [comment]
Most accurate. I believe we found some issues with colors not being converted correctly/accurately with ImageMagick's `convert` command?

**Comment by nullishamy** [comment]
Fair enough. I'm not biased in any particular way so that reason seems good as any. Would also mean that we are consistent org-wide, which is nice.

Just need to think about where this would go. After that I'd be willing to PR some docs in.
